src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -990,19 +1057,1174 @@ partial class C
+
+            comp.VerifyEmitDiagnostics(
+                // (8,28): error CS8663: Both partial member declarations must be readonly or neither may be readonly
+                //     public partial int P { get => throw null!; readonly set { } }
+                Diagnostic(ErrorCode.ERR_PartialMemberReadOnlyDifference, "get").WithLocation(8, 28),
+                // (8,57): error CS8663: Both partial member declarations must be readonly or neither may be readonly
+                //     public partial int P { get => throw null!; readonly set { } }
+                Diagnostic(ErrorCode.ERR_PartialMemberReadOnlyDifference, "set").WithLocation(8, 57));
+        }
+
+        [Fact]
+        public void Accessibility_ExplicitDefault()
+        {
+            // properties can explicitly specify the default accessibility
+            var source = """
+                using System;
+
+                partial class C
+                {
+                    private partial int P1 { get; }
+
+                    static void Main()
+                    {
+                        Console.Write(new C().P1);
+                    }
+                }
+
+                partial class C
+                {
+                    private partial int P1 { get => 1; }
+                }
+                """;
+
+            var verifier = CompileAndVerify(source, expectedOutput: "1");
+            verifier.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void Accessibility_ExplicitDefault_IsRespected()
+        {
+            var source = """
+                using System;
+                
+                class Program
+                {
+                    static void Main()
+                    {
+                        Console.Write(new C().P1); // 1
+                    }
+
+                }
+
+                partial class C
+                {
+                    private partial int P1 { get; }
+                }
+
+                partial class C
+                {
+                    private partial int P1 { get => 1; }
+                }
+                """;
+
+            var comp = CreateCompilation(source);
+            comp.VerifyEmitDiagnostics(
+                // (7,31): error CS0122: 'C.P1' is inaccessible due to its protection level
+                //         Console.Write(new C().P1); // 1
+                Diagnostic(ErrorCode.ERR_BadAccess, "P1").WithArguments("C.P1").WithLocation(7, 31));
+
+            var p1Def = comp.GetMember<SourcePropertySymbol>("C.P1");
+            Assert.True(p1Def.IsPartialDefinition);
+            Assert.Equal(Accessibility.Private, p1Def.DeclaredAccessibility);
+
+            var p1DefPublic = p1Def.GetPublicSymbol();
+            Assert.Equal(Accessibility.Private, p1DefPublic.DeclaredAccessibility);
+        }
+
+        [Fact]
+        public void ModifierDifference_Accessibility_ExplicitDefault_01()
+        {
+            // only one part explicitly specifies the default accessibility
```

Comment:
Do we have a similar test (default accessibility vs. explicit accessibility) for accessors?

Reply:
I think test `ModifierDifference_Accessibility_ExplicitDefault_02` is exercising the scenario adequately because it shows that:
1. explicitly specifying the default accessibility on an accessor is an error
2. a difference in accessor accessibility modifiers between parts is an error